cmd/commands/runtime.go

```diff
@@ -342,7 +342,13 @@ func runtimeInstallCommandPreRunHandler(cmd *cobra.Command, opts *RuntimeInstall
 
 	inferProviderFromRepo(opts.InsCloneOpts)
 
-	err = ensureGitToken(cmd, opts.InsCloneOpts, true)
+	handleValidationFailsWithRepeat(func() error {
+		err = ensureGitToken(cmd, opts.InsCloneOpts, true)
+		if isValidationError(err) {
```

Comment:
what happens in silent mode? I think this is a bug. This only makes sense in case git token is from user input. you should move this to be validation of user input like runtime name and ingress host

Reply:
Didnt understand about silent mode